container/docker/handler.go

```diff
@@ -184,6 +182,10 @@ func (self *dockerContainerHandler) GetSpec() (info.ContainerSpec, error) {
 	}
 
 	spec := libcontainerConfigToContainerSpec(libcontainerConfig, mi)
+
+	ctnr, _ := self.client.InspectContainer(self.id)
+	spec.HasNetwork = runconfig.NetworkMode(ctnr.HostConfig.NetworkMode).IsBridge()
```

Comment:
Nice!

Another possible option is to check if the libcontainer config has an empty [Network](https://github.com/docker/libcontainer/blob/master/configs/config.go#L99). This would save us the Docker inspect call in every `GetSpec()` which would be nice. If that's possible, let's do that :)